crates/ironrdp-cliprdr/src/lib.rs

```diff
@@ -4,3 +4,197 @@
 //! - Clipboard SVC processing (TODO)
 
 pub mod pdu;
+
+use ironrdp_pdu::{decode, gcc::ChannelName, PduEncode, PduResult};
+use ironrdp_svc::{impl_as_any, ChannelPDUFlags, CompressionCondition, StaticVirtualChannel, SvcMessage};
+use pdu::{
+    Capabilities, ClientTemporaryDirectory, ClipboardFormat, ClipboardGeneralCapabilityFlags, ClipboardPdu,
+    ClipboardProtocolVersion, FormatListResponse,
+};
+
+use crate::pdu::FormatList;
+use thiserror::Error;
+use tracing::{error, info};
+
+pub mod clipboard_format {
+    pub const CF_TEXT: u32 = 1;
+    pub const CF_UNICODE: u32 = 13;
+}
```

Comment:
Some info on where these come from could be helpful in case you want to support more formats in the future. Teleport's code for this is [here](https://github.com/gravitational/teleport/blob/36f3a623827dfe0277fa6561d9f2fd2c852fe935/lib/srv/desktop/rdp/rdpclient/src/cliprdr.rs#L712-L747)